CRM/MembershipExtras/Upgrader.php

```diff
@@ -168,13 +168,65 @@ private function removeCustomExternalIDs() {
     }
   }
 
+  /**
+   * Add Related Payment Plan Periods' Custom Fields
+   */
+  private function createPeriodLinkCustomFields() {
+    civicrm_api3('CustomGroup', 'create', [
+      'name' => 'related_payment_plan_periods',
+      'title' => E::ts('Related Payment Plan Periods'),
+      'extends' => 'ContributionRecur',
+      'style' => 'Inline',
+      'collapse_display' => 1,
+      'weight' => 10,
+      'is_active' => 0,
+      'table_name' => 'civicrm_value_related_payme_21',
```

Comment:
I'm not sure why you chose this table name, I guess you created the custom group using either the API or CiviCRM and this was the name generated the system. Thing is if we define the name of the table here and the xml, we normally try to give it a more meaningful name, eg. `civicrm_value_plan_period_linking` or something like that.

After you change the table's name, remember to update the xml also!